pkg/lib/authenticationflow/declarative/intent_account_linking.go

```diff
@@ -142,7 +142,7 @@ func (i *IntentAccountLinking) ReactTo(ctx context.Context, deps *authflow.Depen
 		if err != nil {
 			return nil, err
 		}
-		return NewNodeDoCreateIdentityReactToResult(ctx, deps, NodeDoCreateIdentityOptions{
+		return NewNodeDoCreateIdentityReactToResult(ctx, flows, deps, NodeDoCreateIdentityOptions{
```

Comment:
The convention of the order of the argument should be `ctx, deps, flows` Please change them all.